tavern/util/loader.py

```diff
@@ -82,13 +90,54 @@ def construct_include(loader, node):
     if extension not in ('yaml', 'yml'):
         raise BadSchemaError("Unknown filetype '{}'".format(filename))
 
+    return filename
+
+
+def construct_include(loader, node):
+    """Include file referenced at node."""
+
+    filename = _get_yaml_filename(loader, node)
     with open(filename, 'r') as f:
         return yaml.load(f, IncludeLoader)
 
+def construct_load(loader, node):
```

Comment:
Is there a reason that you put the `loads` block as a separate thing to `includes`? In 'config' files there is a `variables` block which was there so that it could be extended in future (such as by adding `stages`), is there anything getting in the way of reusing `includes`?

This also has the advantage of being able to move the 'login' stage to a config file which can then be swapped out depending on which server is being tested against. eg:

```yaml
stages:
  - !stage login
  - name: get all users
    ...
```
Then the login stage can automatically be swapped with
```shell
$ pytest --tavern-global-cfg test-local.tavern.yaml
$ pytest --tavern-global-cfg test-integration.tavern.yaml
```

Reply:
No. My initial thought was to just reuse ```includes```, but opted for this approach to minimize impact on existing functionality.

However, I think the actual "login stage" should be the same no matter the environment. One can/should still use the variables like in your example to parameterize the different server (or whatever is different between environments).